metricbeat/module/nats/_meta/fields.yml

```diff
@@ -13,7 +13,3 @@
           type: keyword
           description: >
             The server ID
-        - name: server.time
```

Comment:
We may want to keep this in existing modules to avoid breaking changes, though I don't think this field is going to be missed.